File: same-code-diff-code (rozpisanie funkcjonalnosci kodów)/data_structures.py

# Python Data Structures
# deklaruje nazwę listy, 
# w nawiasach kwadratowych po przecinku wypisujemy elementy listy
numbers = [1, 2, 3, 4, 5] # List

# słownik tworzy strukturę klucz-wartość
# w momencie gdy zapytamy o konkretny klucz, w odpowiedzi otrzymamy jego wartość
person = {
  # ustanowienie pary klucz - wartość. Kolejne oddzielone są przecinkiem
  "firstName": "John",
  "lastName": "Doe",
  "age": 30
} # Dictionary


print(person["firstName"]) # Accesses and displays the firstName key of the person dictionary

print(numbers) # Displays the list
print(numbers[0]) # Wyświetlenie 1 elementu tablicy, w tablicy 1 element ma indeks 0
print(numbers[1]) # Wyświetlenie 2 elementu tablicy

# przypisanie nowej wartości (10) do 1 elementu w tablicy, czyli indeksu 0
numbers[0] = 10
print(numbers[0])

File: same-code-diff-code/DataStructures.java

// Java Data Structures
import java.util.ArrayList;
import java.util.HashMap;

public class DataStructures {
    public static void main(String[] args) {
        ArrayList<Integer> numbers = new ArrayList<Integer>();
        numbers.add(1); numbers.add(2); numbers.add(3); numbers.add(4); numbers.add(5);
        HashMap<String, Object> person = new HashMap<String, Object>();
        person.put("firstName", "John");
        person.put("lastName", "Doe");
        person.put("age", 30);

        System.out.println(numbers); // Displays the ArrayList
        System.out.println(person.get("firstName")); // Accesses and displays the firstName key of the person HashMap
    }
}
